RodiSales87/pta-squad-drigo
# Feat/game details

### What I did

- Added toggle functionality to dynamically show/hide "Player 4" in the participants list.
- Updated button styles and logic to reflect the current state (join/leave).
- Interface added to Card

### How to test

1. Open the terminal and navigate to the project folder.
2. In terminal:

  - `git checkout develop`
  - `git pull`
  - `git checkout feat/game-details`
  - `git pull origin feat/game-details`
  - `npm install`
  - `code .`
  - `npm run dev`

3. In your browser, go to http://localhost:3000/.
4. Test the "Enter/Leave" button:

  - Clicking the button should toggle "Player 3" in the participants list.
  - Verify functionality with different numbers of participants and maximum capacity.
